src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4861,6 +4872,35 @@ namespace Akka.Util
         public Akka.Actor.Envelope Peek() { }
         public override string ToString() { }
     }
+    public sealed class MessageBuffer : System.Collections.Generic.IEnumerable<System.ValueTuple<object, Akka.Actor.IActorRef>>, System.Collections.Generic.IReadOnlyCollection<System.ValueTuple<object, Akka.Actor.IActorRef>>, System.Collections.IEnumerable
```

Comment:
Any reason why MessageBuffer should be exposed as a public API?

Reply:
i think it's also public in scala, But can change it to internal

Reply:
changed to internal